cmake/CMakeLists.txt

```diff
@@ -1288,34 +1288,6 @@ if (onnxruntime_USE_OPENVINO)
 
   add_definitions(-DUSE_OPENVINO=1)
 
-  if (EXISTS "$ENV{INTEL_OPENVINO_DIR}/deployment_tools/inference_engine/version.txt")
-    file(READ $ENV{INTEL_OPENVINO_DIR}/deployment_tools/inference_engine/version.txt VER)
-  endif()
-
-  if (NOT DEFINED ENV{INTEL_OPENVINO_DIR})
-    message(FATAL_ERROR "[Couldn't locate OpenVINO] OpenVINO may not have been initialized")
-  endif()
-
-  # Check OpenVINO version for support
```

Comment:
The openvino Version is critical in get_capability of OVEP. It will break the execution flow. Its not advisable to remove the OPENVINO_VERSION

Reply:
See changes in `onnxruntime/core/providers/openvino/ov_versions/capability.cc` where OpenVINO version is extracted from OpenVINO headers. No needs to weird cmake logic to pass OpenVINO version via cmake